src/test/shell/integration/exec_group_test.sh

```diff
@@ -297,6 +297,106 @@ EOF
   grep "test_value" out.txt || fail "Did not find the test-action value"
 }
 
+function test_starlark_test_has_test_execgroup_by_default() {
+  local -r pkg=${FUNCNAME[0]}
+  mkdir $pkg || fail "mkdir $pkg"
+
+  if "$is_windows"; then
+    script_name="test_script.bat"
+    script_content="@echo off\necho hello\n"
+  else
+    script_name="test_script.sh"
+    script_content="#!/bin/bash\necho hello\n"
+  fi
+
+  cat > ${pkg}/defs.bzl <<EOF
+def _impl(ctx):
+    out_file = ctx.actions.declare_file("$script_name")
+    ctx.actions.write(out_file, "$script_content", is_executable=True)
+    return [DefaultInfo(executable = out_file)]
+
+starlark_test = rule(
+    implementation = _impl,
+    test = True,
```

Comment:
Add a comment here that the "test" exec group exists by default in "test" rules.

Reply:
Done (I thought it was enough that the test name said it).